src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,18 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+/**
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {
+
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+     throws IOException, ServletException {
+
+   response.getWriter().println("<h1>RegisterServlet GET request.</h1>");
+ }
+}
```

Comment:
This looks like they want you to add a newline?

Reply:
Oh, gotcha. But I think it's  not necessary anymore, because I modified it in the next step.